assets/dev/js/editor/components/template-library/commands-data/templates.js

```diff
@@ -0,0 +1,9 @@
+import CommandData from 'elementor-api/modules/command-data';
+
+export class Templates extends CommandData {
+	static getEndpointFormat() {
+		return 'template-library/templates';
```

Comment:
The command is called 'templates' and the endpoint is 'template-library'. when you remove cache it is only 'library'. why?

Reply:
Removing the whole component data, since the component name is 'library'.

Reply:
Yes, Please see the next file in the 'Files Changed':
![image](https://user-images.githubusercontent.com/10234691/127632957-0239a827-d644-48ad-94a0-46d85af5830f.png)